src/coreclr/vm/interpreter.cpp

```diff
@@ -9271,19 +9271,6 @@ void Interpreter::DoCallWork(bool virtualCall, void* thisArg, CORINFO_RESOLVED_T
             break;
         }
 
-        // Plus some other calls that we're going to treat "like" intrinsics...
-        if (methToCall == CoreLibBinder::GetMethod(METHOD__STUBHELPERS__SET_LAST_ERROR))
```

Comment:
Is it helpful to add a TODO, like below, for where LastError logic should be reimplemented?

Reply:
I think the correct way to do it would be to save/restore last error using `BEGIN_PRESERVE_LAST_ERROR` macro around any calls out from the interpreter that need to preserve the last error. It is what we do in number of places today. The changes required would not be anywhere near this place.